CS/model/com/vehicledekho/model/Vehicle.java

```diff
@@ -0,0 +1,132 @@
+package com.vehicledekho.model;
+
+public class Vehicle {
+	private String make;
+	private String model;
+	private int engineInCc;
+	private int fuelCapacity;
+	private double mileage;
+	private double price;
+	private double roadTax;
+
+	public String getMake() {
+		return make;
+	}
+
+	public void setMake(String make) {
+		this.make = make;
+	}
+
+	public String getModel() {
+		return model;
+	}
+
+	public void setModel(String model) {
+		this.model = model;
+	}
+
+	public int getEngineInCC() {
+		return engineInCc;
+	}
+
+	public void setEngineInCC(int engineInCC) {
+		this.engineInCc = engineInCC;
+	}
+
+	public int getFuelCapacity() {
+		return fuelCapacity;
+	}
+
+	public void setFuelCapacity(int fuelCapacity) {
+		this.fuelCapacity = fuelCapacity;
+	}
+
+	public double getMileage() {
+		return mileage;
+	}
+
+	public void setMileage(double milage) {
+		this.mileage = milage;
+	}
+
+	public double getPrice() {
+		return price;
+	}
+
+	public void setPrice(double price) {
+		this.price = price;
+	}
+
+	public double getRoadTax() {
+		return roadTax;
+	}
+
+	public void setRoadTax(double roadTax) {
+		this.roadTax = roadTax;
+	}
+
+	public double calculateOnRoadPrice() {
```

Comment:
If this method is override in all subclasses, then do we create this method as abstract and make the class also abstract?